src/lager_transform.erl

```diff
@@ -136,28 +136,41 @@ add_function_transforms(Line, DefaultAttrs, [{Atom, on_log, {Module, Function}}|
     add_function_transforms(Line, {cons, Line, NewFunction, DefaultAttrs}, Remainder).
 
 
+build_dynamic_attrs(Line) ->
```

Comment:
Should these go after `do_transform`? Wasn't paying enough attention to the current convention.